program/split-string-into-array-of-words/SplitStringIntoArrayOfWords.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class SplitStringIntoArrayOfWords {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Enter the String: ");
+        String str = sc.nextLine();
+        System.out.println(Arrays.toString(str.split(" ")));
+    }
+}
```

Comment:
It should return a string array

Reply:
String.str(" "); gives a STRING ARRAY only! 
For better understanding I'll edit the code again. 
Thank you sir.